trunk/problems/POISSON/GALERKIN/my_paraview_driver.F90

```diff
@@ -27,11 +27,20 @@ subroutine my_paraview_driver(IParAttr)
    PARAVIEW_DUMP_GEOM = .true.
 !
 !..load files for visualization upscale
-   if (.not. initialized) then
-      call load_vis(TETR_VIS,trim(FILE_VIS)//'/tetra_'//trim(VLEVEL),TETR)
-      call load_vis(PRIS_VIS,trim(FILE_VIS)//'/prism_'//trim(VLEVEL),PRIS)
-      call load_vis(HEXA_VIS,trim(FILE_VIS)//'/hexa_'//trim(VLEVEL),BRIC)
-      initialized = .true.
+   if (SECOND_ORDER_VIS) then
```

Comment:
I think the better way would be to switch the branches here, i.e.
```
if (.not. initialized) then
   if (SECOND_ORDER_VIS) then
      ...
   else
      ...
   endif
   initialized = .true.
endif
``

Reply:
Change Implemented